e2e/pierced-react-remix-fragment/app/root.tsx

```diff
@@ -49,6 +59,16 @@ export function Layout({ children }: { children: React.ReactNode }) {
 			{children}
 			<ScrollRestoration />
 			<Scripts />
+			{/* 
+          This is a hack to work around an issue with WritableDOM not populating
+          the contents of the last script element (which just so happens to be responsible
+          for hydrating the application). We noticed that if that script tag wasn't the last
+          child of the body, its contents were inserted correctly. We're adding this empty
+          div here to make the script tag before it no longer be the last child.
+
+          TODO: https://jira.cfdata.org/browse/DEVDASH-79
+        */}
+			<div />
```

Comment:
as above

also if this is really needed I would change this so that we would not duplicate the comment twice

e.g. something like:
```tsx
/* the comment on why the hack is needed*/
const HackDiv = () => <div />;
```

and then just use:
```tsx
<HackDiv />
```
in the two jsx results